language-plutus-core/test/Evaluation/Golden/verifySignatureError.plc.golden

```diff
@@ -1 +1 @@
-(error (all a (type) (fun a (fun a a))))
\ No newline at end of file
+Failure
```

Comment:
This is due to the `(<|)` -> `(|>)` change from the above. Now we return a proper `Failure` instead of `EvaluationSuccess (Error () ...)` which was nonsense.